MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,566 @@
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+footerToolbar.Orientaion = ToolbarOrientation.Vertical;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Position
+
+The toolbar can be positioned before or after the image of the ImageEditor by using the `Position` property in the `ImageEditorToolbar` class. The default value is `ToolbarPosition.Start`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+footerToolbar.Position = ToolbarPosition.Start;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar size customization
+
+The size of the toolbar can be adjusted by using the `Size` property in the `ImageEditorToolbar` class.
+
+For horizontal orientation, the size will be considered as width, and for vertical orientation, it will be considered as height.
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image.png");
+    imageEditor.ToolbarSettings.Background = Brush.LightGreen;
+    imageEditor.Toolbars[0].Size = 80;
+    imageEditor.Toolbars[1].Size = 50;
+
+{% endhighlight %}
+
+## Appearance customization
+
+You can customize the appearance of the toolbar by using the `ToolbarSettings` property. Below are some of the properties you can customize:
+
+* `Background`: Specifies background color of the toolbar.
+* `IconColor`: Specifies the color of the icons in the toolbar.
+* `IconSize`: Specifies the size of the icons in the toolbar. The default value is 48.
+* `DisabledColor`: Specifies the color of the disabled icons in the toolbar.
+
+{% tabs %}
+
+{% highlight XAML %}
+  
+<imageEditor:SfImageEditor Source="image.png">
+    <imageEditor:SfImageEditor.ToolbarSettings>
+        <imageEditor:ImageEditorToolbarSettings Background="LightGreen"
+                                                IconColor="Tomato"
+                                                DisabledColor="Lavender"
+                                                IconSize="50, 50" />
+    </imageEditor:SfImageEditor.ToolbarSettings>
+</imageEditor:SfImageEditor>
+      
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+imageEditor.ToolbarSettings.Background = Brush.pink;
+imageEditor.ToolbarSettings.IconColor = Colors.White;
+imageEditor.ToolbarSettings.DisabledColor = Colors.Grey;
+imageEditor.ToolbarSettings.IconSize = new Size(50, 50);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Notifying Toolbar Item Selected
+
+Whenever you tap the toolbar menu item, the `ToolbarItemSelected` event will be triggered, and you can get the respective tapped toolbar item as an argument as shown in the following code snippet. 
+
+Default toolbar item action can be restricted by setting `e.cancel` to `true`.
```

Comment:
The default toolbar item action can be restricted by setting the `e.cancel` to `true.`

Reply:
Updated